src/app/atoms/button/che-button.component.html

```diff
@@ -1,4 +1,7 @@
-<button type="button" class="btn btn{{ outline ? '-outline' : ''}}-{{color}} btn-{{size}} ml-1 mb-1"
-        [disabled]=disabled>
-    <ng-content></ng-content>
-    {{label}}</button>
+<div id="cheButton">
```

Comment:
The ID should be unique. In this case, in each component instance, the id was the same.